Client/Frontend/Fakespot/Views/NoAnalysisCardView.swift

```diff
@@ -0,0 +1,153 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+import Common
+import Shared
+import ComponentLibrary
+
+class NoAnalysisCardViewModel {
+    let cardA11yId: String
+    let headlineLabelText: String
+    let headlineLabelA11yId: String
+    let bodyLabelText: String
+    let bodyLabelA11yId: String
+    let footerLabelText: String
+    let footerLabelA11yId: String
+    var onTapStartAnalysis: (() -> Void)?
+
+    init(cardA11yId: String,
+         headlineLabelText: String,
+         headlineLabelA11yId: String,
+         bodyLabelText: String,
+         bodyLabelA11yId: String,
+         footerLabelText: String,
+         footerLabelA11yId: String) {
+        self.cardA11yId = cardA11yId
+        self.headlineLabelText = headlineLabelText
+        self.headlineLabelA11yId = headlineLabelA11yId
+        self.bodyLabelText = bodyLabelText
+        self.bodyLabelA11yId = bodyLabelA11yId
+        self.footerLabelText = footerLabelText
+        self.footerLabelA11yId = footerLabelA11yId
+    }
+}
+
+final class NoAnalysisCardView: UIView, ThemeApplicable {
+    private struct UX {
+        static let noAnalysisImageViewSize: CGFloat = 104
+        static let headlineLabelFontSize: CGFloat = 15
+        static let bodyLabelFontSize: CGFloat = 13
+        static let footerLabellFontSize: CGFloat = 13
+        static let contentStackViewSpacing: CGFloat = 8
+        static let contentStackViewPadding: CGFloat = 16
+    }
+
+    private var viewModel: NoAnalysisCardViewModel?
+
+    private lazy var cardContainer: ShadowCardView = .build()
+
+    private lazy var contentStackView: UIStackView = .build { stackView in
+        stackView.axis = .vertical
+        stackView.spacing = UX.contentStackViewSpacing
+    }
+
+    private lazy var noAnalysisImageView: UIImageView = .build { imageView in
+        imageView.image = UIImage(named: ImageIdentifiers.shoppingNoAnalysisImage)
+        imageView.contentMode = .scaleAspectFit
+    }
+
+    private lazy var headlineLabel: UILabel = .build { label in
+        label.numberOfLines = 0
+        label.adjustsFontForContentSizeCategory = true
+        label.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .headline,
+                                                            size: UX.headlineLabelFontSize,
+                                                            weight: .bold)
+    }
+
+    private lazy var bodyLabel: UILabel = .build { label in
+        label.numberOfLines = 0
+        label.adjustsFontForContentSizeCategory = true
+        label.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .body,
+                                                            size: UX.bodyLabelFontSize)
+    }
+
+    private let footerLabel: UILabel = .build { label in
```

Comment:
This should be a button instead of a label. This gives us a lot of accessibility features for free which you would have to implement otherwise.